server/routes/comments.js

```diff
@@ -17,15 +25,19 @@ router.get('/:snippetId/comments', function (req, res) {
 });
 
 /* POST new snippet comment*/
-router.post('/:snippetId/comments', function (req, res) {
+router.post('/:snippetId/comments', authChecker, function (req, res) {
   var body = req.body;
   var attributes = {
     content: body.content,
-    SnippetId: req.params.snippetId
+    SnippetId: req.params.snippetId,
+    UserId: 1
   };
   models.Comment.create(attributes)
   .then(function (comment) {
-    res.status(201).send(comment.toJson());
+    models.Comment.findById(comment.id, {include: [models.User]}).then(function (c) {
```

Comment:
@vrael560 my proposition is to create scope `withUser` which one will be including comment author